### DISCUSSION TITLE
Monitoring a mic while listening to audio through a DAC in Ventura

### DISCUSSION THREAD

**Discussion by jayaw** [ExistentialAudio/BlackHole#710] [Q&A]
Hi there - I'm trying to bring just a little bit of my mic input (a Rode USB mic) into my headphones when I'm in a zoom call or recording a screencast etc. I know I can plug my headphones into the mic to get that, but I want to use blackhole so I don't have to do that.

I've got the aggregate device setup here, but I'm not sure how to create that situation described above. I assume when it's active in sound setup the mic would always be hot and giving me just a little bit of that sound input (to my output).

Thanks for any assistance!
<img width="900" alt="image" src="https://user-images.githubusercontent.com/11414533/224395650-fb32d7b4-11e5-4795-9ff5-765c489713f6.png">

**Comment by devinroth** [comment]
Not exactly sure what you’re asking.

You want to be able to hear yourself in your headphones? Why not use GarageBand to with input monitoring to monitor your mic?
